expressEngine/cms/customer/error.js

```diff
@@ -1,3 +1,4 @@
 export default {
     alreadyRegistered: 'You have already registered',
+    unableToDelete: 'Customer not found'
```

Comment:
replace `unableToDelete: 'Customer not found'` to `customerNotFound: 'Customer not found'`
Reason: Try to keep variable names meaningful.

Reply:
Done with the change.